firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Endian.txt

```diff
@@ -0,0 +1,12 @@
+SCENARIO("read endian data")
+  GIVEN("A buffer '\x01\x0f'")
+  WHEN("read buffer 2UL ")
+     THEN("reads correct value")
```

Comment:
I would suggest stating the actual correct values, e.g. `0x010f`, to make clear how the bytes should be arranged in the outputs